blobs/howto/python/python-v12/crud_datalake.py

```diff
@@ -121,15 +121,15 @@ def initialize_storage_account(storage_account_name, storage_account_key):
 # -------------------------------------------------
 
 # <Snippet_AuthorizeWithAAD>
```

Comment:
We could also update this snippet to match the pattern for blobs. Something like:

```
def initialize_storage_account_ad(self, storage_account_name):
    account_url = f"https://{storage_account_name}.dfs.core.windows.net"
    default_credential = DefaultAzureCredential()

    # Create the DataLakeServiceClient object
    service_client = DataLakeServiceClient(account_url, credential=default_credential)

    return service_client
```

Reply:
Done